editor/animation_track_editor.cpp

```diff
@@ -4040,41 +4042,39 @@ void AnimationTrackEditor::insert_node_value_key(Node *p_node, const String &p_p
 
 	path += ":" + p_property;
 
-	NodePath np = path;
```

Comment:
Why did you remove it? Comparing NodePath with String is slower.

Reply:
My bad. It looked redundant at first glance. I've restored it now.